ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/UpdateMutabilityInSignatureFixTest.fs

```diff
@@ -0,0 +1,12 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
+type UpdateMutabilityInSignatureFixTest() =
+    inherit FSharpQuickFixTestBase<UpdateMutabilityInSignatureFix>()
+    override x.RelativeTestDataPath = "features/quickFixes/updateMutabilityInSignatureFix"
```

Comment:
Nit-picking: could you please add empty lines after `inherit` and `override` members?